psydac/api/tests/test_assembly.py

```diff
@@ -168,4 +181,4 @@ def test_non_symmetric_BilinearForm(backend):
     test_field_and_constant(None)
     test_multiple_fields(None)
     test_math_imports(None)
-    test_non_symmetric_BilinearForm(None)
+    test_non_symmetric_BilinearForm('python')
```

Comment:
This part of the file is only here as a convenience to developers, and it is not run by Pytest. Is it important to make this change?